rudder-webapp/debian/preinst

```diff
@@ -46,10 +46,8 @@ case "$1" in
         /etc/init.d/rudder-jetty stop >&2 > /dev/null || true
       fi
 
-      if [ -x /opt/rudder/bin/rudder-pkg ]
-      then
-        /opt/rudder/bin/rudder-pkg plugin save-status > /tmp/rudder-plugins-upgrade
-      fi
+      rudder package plugin save-status > /tmp/rudder-plugins-upgrade
```

Comment:
I'm not sure it will be there in preinst when upgrading from 5.0

Reply:
ha yes